packages/vue/src/components/organisms/SfProductCardHorizontal/SfProductCardHorizontal.vue

```diff
@@ -0,0 +1,301 @@
+              class="sf-product-card-horizontal__picture"
+              :src="picture"
+              :alt="title"
+              :width="imageWidth"
+              :height="imageHeight"
+            />
+          </template>
+          <SfImage
+            v-else
+            class="sf-product-card-horizontal__image"
+            :src="image"
+            :alt="title"
+            :width="imageWidth"
+            :height="imageHeight"
+          />
+        </slot>
+      </div>
+    </component>
+    <div class="sf-product-card-horizontal__main">
+      <div class="sf-product-card-horizontal__header">
+        <component
+          :is="linkComponentTag"
+          :href="linkComponentTag === 'a' ? link : undefined"
+          :to="link && linkComponentTag !== 'a' ? link : undefined"
+          class="sf-product-card-horizontal__link"
+        >
+          <slot name="title" v-bind="{ title }">
+            <h3 class="sf-product-card-horizontal__title">
+              {{ title }}
+            </h3>
+          </slot>
+        </component>
+        <slot name="price" v-bind="{ specialPrice, regularPrice }">
+          <SfPrice
+            v-if="regularPrice"
+            class="sf-product-card-horizontal__price"
+            :regular="regularPrice"
+            :special="specialPrice"
+          />
+        </slot>
+      </div>
+      <button
+        v-if="wishlistIcon !== false"
+        :aria-label="ariaLabel"
+        :class="wishlistIconClasses"
+        class="mobile-only"
+        @click="toggleIsOnWishlist"
+      >
+        <slot name="wishlist-icon" v-bind="{ currentWishlistIcon }">
+          <SfIcon
+            :icon="currentWishlistIcon"
+            color="black"
+            size="19px"
+            data-test="sf-wishlist-icon"
+          />
+        </slot>
+      </button>
+      <div class="sf-product-card-horizontal__details">
+        <slot name="description">
+          <p class="sf-product-card-horizontal__description desktop-only">
+            {{ description }}
+          </p>
+        </slot>
+        <slot name="configuration"></slot>
+        <slot name="reviews" v-bind="{ maxRating, scoreRating }">
+          <div
+            v-if="typeof scoreRating === 'number'"
+            class="sf-product-card-horizontal__reviews"
+          >
+            <SfRating
+              class="sf-product-card-horizontal__rating"
+              :max="maxRating"
+              :score="scoreRating"
+            />
+            <a
+              v-if="reviewsCount"
+              class="sf-product-card-horizontal__reviews-count"
+              href="#"
+              @click="$emit('click:reviews')"
+            >
+              {{ reviewsCount }} reviews
+            </a>
+          </div>
+        </slot>
+      </div>
+      <div class="sf-product-card-horizontal__actions desktop-only">
+        <slot name="actions"></slot>
+        <slot name="add-to-cart">
+          <SfAddToCart
+            v-model="qty"
```

Comment:
check this v-model, imo it't doesn't work :) 
to test it you can
1. set initial state by 2
2. log this value after click

check how it works collected product :) and it SfQuantitySelector :)